WMF Framework/Significant Events Endpoint/ArticleAsLivingDocViewModels.swift

```diff
@@ -249,13 +252,19 @@ public struct ArticleAsLivingDocViewModel {
 
         var newlyCollapsedSectionHashes: [Int] = []
 
+        // Returns the small event collapsed section that represents the `sectionHash`, if one exists
+        func firstCollapsedSectionContaining(sectionHash: Int) -> CollapsedSection? {
+            return collapsedSections
+                .first { collapsedSection in collapsedSection.sectionHashes.compactMap { $0 }
```

Comment:
Do we need the compactMap here? It just pulls out any nils, but sinc sectionHash can never be nil we don't need to worry about that? (This already was in the code, I just didn't catch it before.)

Reply:
Good catch - updated to a plain ole `map`.

Reply:
Why do we need to use `map`... can it just be 

`return collapsedSections.first { collapsedSection in collapsedSection.sectionHashes.contains(sectionHash) }`

or even (depending on personal preference)

`return collapsedSections.first { $0.sectionHashes.contains(sectionHash) }`

Reply:
Haha oops – this officially means no more programming for me today 🥴. Updated – now permanently shutting down my computer forever.

Reply:
We all have those moments sometimes - I certainly have more than my share. :)